packages/runtime/src/runtimes/react/controls/checkbox.ts

```diff
@@ -1,10 +0,0 @@
-import { type DataType, type ResolvedValueType } from '@makeswift/controls'
-
-import { CheckboxDefinition } from '../../../controls'
-
-export function useCheckboxControlValue(
-  data: DataType<CheckboxDefinition> | undefined,
-  definition: CheckboxDefinition,
-): ResolvedValueType<CheckboxDefinition> {
-  return definition.fromData(data) ?? definition.config.defaultValue
```

Comment:
Here and below, per-control `use...ControlValue` hooks were replaced by the `useResolvedValue` hook: https://github.com/makeswift/makeswift/pull/814/files#diff-d4ed6a20b5476c4eb7f48f030d283e619c8ae2b1bf607492e088118237d182fd